Clinical-Genomics/scout
# Fix cancer variants view with cancer-specific dismissed options

fix #2302 

Problem: when a cancer variant is dismissed with a [cancer-specific dismissed option](https://github.com/Clinical-Genomics/scout/blob/1d48b12e47b78d7f7765d0b6506abeb3b4245f80/scout/constants/variant_tags.py#L242), then the cancer variantS page (of the same type of the variant) crashes

**How to test**:
- Locally, load the demo cancer case and dismiss a SNV or SV (or both) with one of the cancer specific reasons
- Go back to the variantS view and notice that it is broken on master branch
- Switch to this branch and the view should work again
- 
**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
